tests/e2e/core-tests/specs/index.js

```diff
@@ -154,4 +156,5 @@ module.exports = {
 	runCartRedirectionTest,
 	runMyAccountCreateAccountTest,
 	runOrderEmailReceivingTest,
+	runInitiateWccomConnectionTest,
```

Comment:
This is having the test run twice. The api tests are ones that test the local install's rest api.

Reply:
Sorry, please ignore this comment.